Components/Alerts.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Metaphor UI | Alerts</title>
+    <link rel="stylesheet" href="./docs.css" />
+    <link rel="stylesheet" href="./MetaphorUI.css" />
+  </head>
+  <body>
+    <!-- TOP NAV BAR -->
+
+    <nav class="navCont">
+      <div class="logCont">
+        <a href="../index.html" class="logoLink"
+          ><h1 class="logoText">
+            Meta<span class="logoTextHigh">phor</span>
+          </h1></a
+        >
+      </div>
+      <div class="navItems">
+        <a href="./docs.html" class="navDocs">Docs</a>
+        <h3 class="navVersion">Ver. 0.0.1</h3>
+      </div>
+      <div class="burgerIcon" onclick="toggleSidebar()">
+        <ion-icon name="menu-outline"></ion-icon>
+      </div>
+    </nav>
+
+    <!-- MAIN CONTAINER -->
+
+    <div class="mainCont">
+      <!-- SIDEBAR CONTAINER -->
+
+      <div class="sidebarCont hideSidebar">
+        <div class="sidebarList">
+          <a class="sidebarListItem" href="docs.html">Getting Started</a>
+          <hr class="hr-dark mt8" />
+          <a class="sidebarListItem" href="./Typography.html">Typography</a>
+          <a class="sidebarListItem" href="./Buttons.html">Buttons</a>
+          <a class="sidebarListItem" href="./Cards.html">Cards</a>
+          <a class="sidebarListItem" href="./Images.html">Images</a>
+          <a class="sidebarListItem" href="./inputs.html">Inputs</a>
```

Comment:
html file names look inconsistent,I am not sure if we should even be using capital letter to start our html file name

Reply:
Thanks for the review Aman, Fixed this issue